examples/flax/language-modeling/run_t5_mlm_flax.py

```diff
@@ -522,9 +523,7 @@ def write_eval_metric(summary_writer, eval_metrics, step):
             model_args.config_name, cache_dir=model_args.cache_dir, vocab_size=len(tokenizer)
         )
     elif model_args.model_name_or_path:
-        config = T5Config.from_pretrained(
-            model_args.model_name_or_path, cache_dir=model_args.cache_dir, vocab_size=len(tokenizer)
-        )
+        config = T5Config.from_pretrained(model_args.model_name_or_path, cache_dir=model_args.cache_dir)
```

Comment:
When loading `config` from pre-trained models, we shouldn't set the `vocab_size` here.
This results in an error as there is a mismatch in `len(tokenizer)` and the `vocab_size` used in the pre-trained T5 models. Which is why right now it's not possible to load a pre-trained model with this script.

Reply:
I am a bit worried though that people will do the same mistake that was made for GPT2 pretraining during the Flax/JAX sprint when a model is trained from scratch with a tokenizer that is larger than the model. In such a case it is possible that training can go for a long time until it errors out. 

On the other hand, I definitely see the need to remove this!

Could we maybe do the following:

remove `len(tokenizer)` here just as you did but then add the following:

```
config.vocab_size = len(tokenizer)
```

under 

```=
model = FlaxT5ForConditionalGeneration(config, seed=training_args.seed, dtype=getattr(jnp, model_args.dtype))
```

which is only called for "from scratch" pretraining? This way we are save for both cases.